packages/docs/content/blog/release-6.0.mdx

```diff
@@ -19,17 +19,17 @@ Previously we had two options for teams consuming our design system styles: CSS
+With this move, [as we mentioned in our previous blog post](/blog/#sasscss-changes), we've decided to stop distributing a Sass version of our stylesheets in our npm package, but we are still providing Sass files with the Sass versions of all our theme variables while teams transition over to using CSS Custom Properties. You can [read about how to use the CSS files and our theme variables](/getting-started/for-developers/) in our getting-started guide for developers. If you're currently using Sass, we've put together a more detailed [Sass to CSS migration guide for you](/migration-guides/sass-to-css/).
 
 ### New masked fields and automatic SSN obfuscation
 
-When we created the [single-input date field](https://design.cms.gov/components/date-field/single-input-date-field/) a while back, we introduced a new masking pattern called the [label-mask](https://design.cms.gov/components/text-field/label-masked-field/). Now that we've been using it internally for a few months, we've decided to open it up for use as a drop-in replacement for our old [input-mask](https://design.cms.gov/components/text-field/input-masked-field/) pattern. The label-mask now supports every feature that the old input-mask supported, with the additional improvement that the Social Security Number (SSN) field now automatically obfuscates the SSN when the input doesn't have focus by only showing the last four digits.
+When we created the [single-input date field](/components/date-field/single-input-date-field/) a while back, we introduced a new masking pattern called the [label-mask](/components/text-field/label-masked-field/). Now that we've been using it internally for a few months, we've decided to open it up for use as a drop-in replacement for our old [input-mask](/components/text-field/input-masked-field/) pattern. The label-mask now supports every feature that the old input-mask supported, with the additional improvement that the Social Security Number (SSN) field now automatically obfuscates the SSN when the input doesn't have focus by only showing the last four digits.
 
 <div className="ds-u-measure--wide ds-u-text-align--center">
   <img src={ssnInput} alt="Animated screen capture of inputting a Social Security Number using the new label-mask pattern" className="ds-u-display--inline-block" />
 </div>
 
-Hiding the SSN is previously something application teams had to implement themselves to use the SSN field and maintain their users' privacy. You can [read more about the label-mask pattern](https://design.cms.gov/components/text-field/label-masked-field/)—what it is and why we built it—on the new documentation page.
+Hiding the SSN is previously something application teams had to implement themselves to use the SSN field and maintain their users' privacy. You can [read more about the label-mask pattern](/components/text-field/label-masked-field/)—what it is and why we built it—on the new documentation page.
```

Comment:
"—what" needs a space I believe

Reply:
Same with "it—on"

Reply:
It's hard to see in the monospace font, but that character is a dash, so there's actually not supposed to be any spaces around it